woocommerce/woocommerce-android
# Shippling lines M1 analytics

Closes: #11392 

### Description
We want to track 3 events around shipping method selection:

* Updated event:
   * `order_shipping_method_add` with the custom property `shipping_method` containing the id of the selected shipping method in lowercase (`ups`|`flat_rate`|`free_shipping`…).
   * Event registration:  https://github.com/Automattic/tracks-events-registration/pull/2406
* New event:
   * `order_shipping_method_selected`, triggered every time a merchant selects a new shipping method.
   * This event contains a custom property `shipping_method` with the id of the selected shipping method in lowercase (`ups`|`flat_rate`|`free_shipping`…).
   * Event registration:  https://github.com/Automattic/tracks-events-registration/pull/2408
* New event:
   * `order_add_shipping_tapped`, triggered every time a merchant taps on the add shipping button.
   * Event registration:  https://github.com/Automattic/tracks-events-registration/pull/2407

### Testing instructions


1. Open the app
2. Go to orders
3. Tap on Create New Order (+)
4. Add a product to enabled the add shipping button
5. Tap on Add shipping
6. Check that the `order_add_shipping_tapped` event is triggered
7. Tap on select shipping method
8. Select a shipping method
9. Check that the `order_shipping_method_selected` event is triggered and contains the `shipping_method` custom property
10. Tap on Add shipping
11. Check that the `order_shipping_method_add` event is triggered and contains the `shipping_method` custom property

### Images/gif

```
🔵 Tracked: order_add_shipping_tapped, Properties: {"blog_id":222408976,"is_wpcom_store":true,"was_ecommerce_trial":true,"plan_product_slug":"wooexpress-small-bundle-yearly","store_id":"b6468f83-4986-44f1-b4ce-6021649dbf29","is_debug":true,"site_url":"https:\/\/woo-practically-automatic-witch.wpcomstaging.com","cached_woo_core_version":"8.8.3"}
🔵 Tracked: order_shipping_method_selected, Properties: {"shipping_method":"free_shipping","blog_id":222408976,"is_wpcom_store":true,"was_ecommerce_trial":true,"plan_product_slug":"wooexpress-small-bundle-yearly","store_id":"b6468f83-4986-44f1-b4ce-6021649dbf29","is_debug":true,"site_url":"https:\/\/woo-practically-automatic-witch.wpcomstaging.com","cached_woo_core_version":"8.8.3"}
🔵 Tracked: order_shipping_method_add, Properties: {"flow":"creation","shipping_method":"free_shipping","blog_id":222408976,"is_wpcom_store":true,"was_ecommerce_trial":true,"plan_product_slug":"wooexpress-small-bundle-yearly","store_id":"b6468f83-4986-44f1-b4ce-6021649dbf29","is_debug":true,"site_url":"https:\/\/woo-practically-automatic-witch.wpcomstaging.com","cached_woo_core_version":"8.8.3"}

```
